net/simple-adblock/Makefile

```diff
@@ -5,8 +5,8 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=simple-adblock
-PKG_VERSION:=1.8.0
-PKG_RELEASE:=4
+PKG_VERSION:=1.8.1
+PKG_RELEASE:=7
```

Comment:
Shouldnt be here 1?

Reply:
No.

Reply:
Thanks for sharing any details about that. Why it shouldn't be there? Is it because of your own repository? Anyway looking at [CONTRIBUTING.md](https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md), which is in this repository and I need to quote it:
> PKG_RELEASE should be initially set to 1 or reset to 1 if the software version is changed.

Reply:
Yep, precisely because of my own repo. I did reset the PKG_RELEASE to 1 when I bumped the version, but has since increased it.